### PR TITLE
Feat backend

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
- Added backend for quiz
- Quiz, its answers and showing results at the end of the quiz

### CODE REVIEW

**Comment by Kothalkarpriya** [comment]
> * Added backend for quiz
> * Quiz, its answers and showing results at the end of the quiz

Deploy Preview: https://629f2088f60aa20008255ab8--quiz-application-by-priya.netlify.app/

File: src/assests/styles/quiz.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  margin: 0.5rem 0;
+  font-size: 1.5rem;
+}
+.ques-head p{
+  margin:10px 0;
+}
+
+.question-list{
+width: 100%;
+padding: 0.5rem;
```
Comment (ShreyashMahajan) [inline src/assests/styles/quiz.css:24]: try to use px for less than 1rem

File: src/backend/controllers/question.controller.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { Response } from 'miragejs';
+
+import {
+  // requiresAuth,
+  shuffleArray,
+} from '../utils';
+
+export const getQuestionsHandler = function (schema, request) {
+  // const user = requiresAuth.call(this, request);
```
Comment (ShreyashMahajan) [inline src/backend/controllers/question.controller.js]: you can remove commented code

**Review by ShreyashMahajan — APPROVED** [review]
code looks well structured and good file strucutre. Good to merge

File: src/assests/styles/colors.css [hunk 3]
```diff
@@ -3,11 +3,13 @@
 :root {
     --primary: #F8F9FA;
     --theme: #212529;
+    --black: black;
```
Comment (SJTGSHIVAM) [inline src/assests/styles/colors.css]: Its always better to use #000 instead of keyword black.

File: src/assests/styles/quiz.css (hunk 1 above)
Comment (Kothalkarpriya) [reply src/assests/styles/quiz.css:24]: Ok

File: src/backend/controllers/question.controller.js (hunk 2 above)
Comment (Kothalkarpriya) [reply src/backend/controllers/question.controller.js]: I will remove it

File: src/assests/styles/colors.css (hunk 3 above)
Comment (Kothalkarpriya) [reply src/assests/styles/colors.css]: ok I will change it

**Review by SJTGSHIVAM — COMMENTED** [review]
overall things look good to merge.

**Pull request merged**